front/src/app/shared/models/shared.models.ts

```diff
@@ -7,15 +7,20 @@ export interface TaskGroupsList {
   lists: TaskGroup[];
 }
 
-export interface Tasks {
+export interface Task {
   id: string;
   title: string;
 }
 
 export interface TaskList {
-  tasks: Tasks[];
+  tasks: Task[];
 }
 
 export interface NewTaskList {
   list: TaskGroup;
 }
+
+export interface TasksGroup {
+  allTasks: Task[];
+  task: Task;

```

Comment:
It's confusing to have `TaskList` and `TasksGroup` interfaces (hard to understand the difference between them).

Additionally, `TasksGroup` name doesn't really represent the content of this interface.

I would suggests naming it as `createTaskResponse` since it doesn't represent any concrete entity, but just a technical construction made in order to pass a bunch of data over http